### PR TITLE
Update Arabic - Added Missing Stuff

### PR DESCRIPTION
**Pull request by Omaranwa**
Applying for issue(s):
- #2005
- #2027
- #2035
- #2104
- #2111
- #2133
- #2140
- #2144
- #2149
- #2158
- #2165
- #2176
- #2181
- #2188
- #2199
- #2205
- #2214 
- #2220
- #2243
- #2251
- #2264
- #2278
- #2284
- #2286
- #2294
- #2300
- #2311
- #2327
- #2328

### CODE REVIEW

File: objects/ar-EG.json [hunk 1]
```diff
@@ -7,42 +7,14 @@
         "reference-name": "Pink Water",
         "name": "مياه وردية"
     },
-    "toontowner.scenery_small.ttrftl07": {
```
Comment (Gymnasiast) [inline objects/ar-EG.json:10]: You accidentally removed these object translations from this file. Could you restore them?

Comment (Omaranwa) [reply objects/ar-EG.json:10]: well, do thanks!

**Comment by Omaranwa** [comment]
@Gymnasiast done, could you pls merge it?

File: data/language/ar-EG.txt [hunk 2]
```diff
@@ -175,6 +176,7 @@ STR_0599    :A compact roller coaster with individual cars and smooth twisting d
 STR_0600    :Powered mine trains career along a smooth and twisted track layout
 STR_0602    :Roller coaster trains are accelerated out of the station by linear induction motors to speed through twisting inversions
 STR_0603    :أفعوانية خشبية مع مسار فولاذي، تسمح بإنحدارت وإنعكاسات حادة.
+STR_0604    :Riders ride single file on a narrow monorail track, as they race through tight inversions and direction changes.
```
Comment (Gymnasiast) [inline data/language/ar-EG.txt:179]: You forgot to translate this sentence.

File: data/language/ar-EG.txt [hunk 3]
```diff
@@ -3942,3 +4042,354 @@ STR_DTLS    :
 STR_SCNR    :حصن مفارقة تاريخية
 STR_PARK    :حصن مفارقة تاريخية
 STR_DTLS    :
+
```
Comment (Gymnasiast) [inline data/language/ar-EG.txt:4045]: You should only add strings to this file if they are translated. If you do not have time to translate them now, it’s better to leave them out - our translation system will fall back to English for those strings automatically.

Comment (Omaranwa) [reply data/language/ar-EG.txt:4045]: What do mean?

File: data/language/ar-EG.txt (hunk 2 above)
Comment (Omaranwa) [reply data/language/ar-EG.txt:179]: ah ok

**Comment by Gymnasiast** [comment]
Merged as https://github.com/OpenRCT2/Localisation/commit/9a7a9fb72bfb6dbf796b0a95c041f90f62423003.

**Pull request closed without merging**